boards/arm/lpcxpresso55s69/lpcxpresso55s69_cpu0.dts

```diff
@@ -136,26 +136,6 @@
 	pinctrl-names = "default";
 };
 
-&dma0 {
-	/*
-	 * The total number of dma channels available is defined by
-	 * FSL_FEATURE_DMA_NUMBER_OF_CHANNELS in the SoC features file.
-	 * Since memory from the heap pool is allocated based on the number
-	 * of DMA channels, set this property to as many channels is needed
-	 * for the platform. Adjust HEAP_MEM_POOL_SIZE in case you need more
-	 * memory.
-	 */
-	dma-channels = <20>;
-	nxp,dma-num-of-otrigs = <4>;
-	status = "okay";
-};
-
-&dma1 {
-	dma-channels = <10>;
-	nxp,dma-num-of-otrigs = <4>;
-	status = "okay";
-};
```

Comment:
We are removing `nxp,dma-num-of-otrigs` as well. Should we delete only `dma-channels`

Reply:
all of these things are just being relocated